crates/bevy_sprite/src/render/mod.rs

```diff
@@ -430,6 +431,7 @@ pub fn extract_sprites(
                     image_handle_id: sprite.image.id(),
                     anchor: sprite.anchor.as_vec(),
                     original_entity: Some(original_entity),
+                    scaling_mode: sprite.image_mode.scale(),
```

Comment:
I'd prefer it if the sprite geometry was calculated during extraction rather than in prepare.

Reply:
All the geometry logic happens in [`prepare`](https://github.com/bevyengine/bevy/pull/17258/files#diff-77918ecd6990bcb870a2d09f505f056ac01f54ce72fba80d692acd0484153810R776-R798) for now. I am going with less resistance and fewer changes in the codebase to achieve the necessary functionality.